htdocs/takepos/invoice.php

```diff
@@ -285,6 +285,8 @@ function fail($message)
 					$payment->create($user);
 					$payment->addPaymentToBank($user, 'payment', '(CustomerInvoicePayment)', $bankaccount, '', '');
 					$remaintopay = $invoice->getRemainToPay(); // Recalculate remain to pay after the payment is recorded
+				} elseif (!empty(getDolGlobalInt("TAKEPOS_DELAYED_TERMS"))) {
```

Comment:
`elseif (!empty(getDolGlobalInt("TAKEPOS_DELAYED_TERMS"))) {`
must be replaced with
`elseif (getDolGlobalInt("TAKEPOS_DELAYED_TERMS")) {`
empty is not allowed of the result of a function

Reply:
@eldy fixed